src/aq.py

```diff
@@ -153,7 +209,7 @@ def get_scales(self) -> torch.Tensor:
         else:  # train scale codebook only
             return self.scales_clusters.gather(1, self.scales_indices)[:, :, None, None]
 
-    def forward(self, selection: Union[slice, ellipsis, torch.Tensor] = ...):
+    def reconstruct_weight(self, selection: Union[slice, ellipsis, torch.Tensor] = ...):
```

Comment:
I'd request that we keep it as "forward" for notebook and experiment compatibility.
Alternatively, 

def forward(self, *args, **kwargs): self.reconstruct_weight(*args, **kwargs)?

Reply:
I would make sense for this layer to behave like a normal `nn.Linear` layer. That is, actually perform a forward pass on `forward`. That way we won't have to change much code when replacing `nn.Linear`s with it.

Reply:
Ah, I get it now. Maybe best to move that to QuantizedLinear class and keep QuantizedWeight as is?

Reply:
Done more or less